### PR TITLE
Version 0.05 Update

### PR DESCRIPTION
**Pull request by puneetsl**
Update the Web version of archive to latest electron and nativefier.

- Now Spell check is supported inherently
- Stability improvements
- All links open within Notion, You can open links in external browser by right-clicking and selecting `open in default browser`

### CODE REVIEW

**Comment by joshpetit** [comment]
I don't see a change to the tar, have you not pushed the new script and tar ball yet?

**Comment by puneetsl** [comment]
I can see Github showing new tar with the PR
image

not sure why it isn't showing to you

**Comment by joshpetit** [comment]
Haha my fault, I completely overlooked it 😂

File: README.md [hunk 1]
```diff
@@ -34,7 +34,7 @@ curl https://raw.githubusercontent.com/puneetsl/lotion/master/setup.sh > setup.s
 chmod +x setup.sh
 
 # Run (with sudo for global installation, without sudo for local installation)
-[sudo] ./setup.sh native
+[sudo] ./setup.sh nativej
```
Comment (joshpetit) [inline README.md:37]: Was this intentionally changed to nativej or just a typo?

**Review by joshpetit — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (puneetsl) [reply README.md:37]: whoops! not sure it ended up there

Comment (puneetsl) [reply README.md:37]: Good catch! I'll fix this

**Pull request merged**